### ISSUE TITLE
Un-used GPU gets hot

### ISSUE THREAD

**Issue by SimonBrazell** [khronokernel/How-to-disable-your-unsupported-GPU-for-MacOS#3]
Hi, thanks for creating this guide, very useful. 

I'm noticing that my GPU gets pretty hot (the fans never seem to spin), even though it is not being used for rendering at all, I'm curious if you or anyone else experiences this also?

**Note -** as I only have a single GPU (1080 ti), I used option 1 `-wegnoegpu`, wonder if this makes a difference at all...

**Comment by ChrisEngelhardt** [comment]
same issue here, any way to avoid this?

**Comment by SimonBrazell** [comment]
Not to my knowledge, I never got a response, perhaps just improve your case cooling?

**Comment by rdziubek** [comment]
> Not to my knowledge, I never got a response, perhaps just improve your case cooling?

Nah it's not a cooling problem, I'm right now trying to make a switch from Clover to OpenCore and what I was only able to achieve with turning off my Fermi Nvidia (820m, dGPU with Haswell) was preventing macOS from seeing it–I can't get it to turn off, whereas I could do so using Clover. Tried both `-wegnoegpu` and the suggested SSDT option with same results. 
I think that the only solution for now is simply to load the modified DSDT (with the `_OFF` method injected into GPU's init method) as a whole, just like it used to be done on Clover systems. Which kind of defeats some of OpenCore principles though.

**Comment by SimonBrazell** [comment]
@witampanstwa are you saying there is a way to completely turn off the un-used GPU in Clover? 

I've used `-wegnoegpu` which does indeed disable the GPU for macOS but it still seems to get hot after the system has been running for some time.

**Comment by rdziubek** [comment]
@SimonBrazell yeah, I mean it is not Clover-specific, but it is as per Clover configuration guidance; it boils down to decompiling and recompiling the DSDT itself, not just SSDT as it is expected to be done in OpenCore configuration. You can find RehabMan's posts regarding this. 
I don't think OpenCore would have problems loading that even though I haven't tried that, you can give it a try and let us know

**Comment by SimonBrazell** [comment]
Oh right, I'll need to read up on that, not had to mess with the DSDT for my setup so far.

**Comment by ChrisEngelhardt** [comment]
I've tried it with the DSDT patch and clover but GPU still gets hot.
Now I'm using OpenCore and disabled the PCI path. The GPU is no more visible to the system but also still gets hot

**Comment by SimonBrazell** [comment]
@ChrisEngelhardt thanks for the feedback, I'll not bother doing the reading then 🙂 seems like this issue is unavoidable.
